openshift/e2e-common.sh

```diff
@@ -272,6 +272,9 @@ function run_e2e_tests(){
   --type 'merge' \
   --patch '{"spec": {"maxReplicas": '${OPENSHIFT_REPLICAS}', "minReplicas": '${OPENSHIFT_REPLICAS}'}}' || return 1
 
+  # TODO: enable autoscaler HA test once https://github.com/knative/operator/issues/337 was solved.
+  rm ./test/ha/autoscaler_test.go
```

Comment:
Can we drop this in the script that updates the repo? That way we won't have to do it again once we pull it into serverless-operator.

Reply:
Sure, that's great idea. Updated.